### ISSUE TITLE
Sort files by last opened instead of last modified?

### ISSUE THREAD

**Issue by jondkinney** [topfunky/PeepOpen-Issues#61]
I am very used to being able to alternate between 4-5 files via cmd+t and then a few down arrow keystrokes with an enter when I reach my desired file. You can't really do that very well with PeepOpen. You can flip between 2 files (which was nice when that was added) but other than those 2 files you have to constantly search for the file you want. If it Peep open could maintain the list in order of last accessed/opened (like I think Textmate does) that would at least make _me_ very happy :-)

Thanks!

**Comment by ghost** [comment]
Definitely agree here. Switching between files in the order they were opened is what makes the "go to file" feature so nice.

**Comment by szich** [comment]
I couldn't agree more.  I've decided to revert to the existing cmd+t command until this feature is added.

**Comment by topfunky** [comment]
The new native (Objective C-based) TextMate plugin in 0.2.1 will help us provide more natural sorting of files, even if they weren't modified.

I can't promise a release date but hope to use the two together to provide the expected behavior for opened vs. modified files.

**Comment by neilsarkar** [comment]
Bump. 

I'm with szich, had to switch back to the default cmd+t. Especially when you're modifying coffeescript or sass that behavior is a dealbreaker for this (otherwise wonderful) plugin. Would love a fix on this!

**Comment by jondkinney** [comment]
Any update on this? I had to ditch PeepOpen almost as soon as I bought it because this feature isn't there and just tried it again tonight but see that a over a year later this planned feature still hasn't made it in :(

**Comment by ghost** [comment]
Agreed, in my opinion this is probably the most important feature. Being able to go to a file quickly is great, but it doesn't come close to how many times I use the recently opened.

**Comment by adevadeh** [comment]
+1 this would greatly improve peepopen. looking forward to it!
